settings.py

```diff
@@ -477,23 +477,25 @@
 OPENROSA_VERSION = "1.0"
 
 # OTA restore fixture generators
-FIXTURE_GENERATORS = [
-    "corehq.apps.fixtures.fixturegenerators.hq_fixtures",
-]
-
-HQ_FIXTURE_GENERATORS = [
-    # core
-    "corehq.apps.users.fixturegenerators.user_groups",
-    "corehq.apps.fixtures.fixturegenerators.item_lists",
-    "corehq.apps.callcenter.fixturegenerators.indicators_fixture_generator",
-    "corehq.apps.products.fixtures.product_fixture_generator",
-    "corehq.apps.programs.fixtures.program_fixture_generator",
-    "corehq.apps.locations.fixtures.location_fixture_generator",
-    # custom
-    "custom.bihar.reports.indicators.fixtures.generator",
-    "custom.m4change.fixtures.report_fixtures.generator",
-    "custom.m4change.fixtures.location_fixtures.generator",
-]
+FIXTURE_GENERATORS = {
+    # fixtures that may be sent to the phone independent of cases
+    'standalone': [
+        # core
+        "corehq.apps.users.fixturegenerators.user_groups",
```

Comment:
Honestly hadn't given it significant thought. I was just laying the ground
work. We could probably draw links to a number of the fixtures.
On 2 Mar 2015 20:56, "Cory Zue" <notifications@github.com> wrote:

> In settings.py
> <https://github.com/dimagi/commcare-hq/pull/5720#discussion_r25626104>:
>
> > -    "corehq.apps.users.fixturegenerators.user_groups",
> > -    "corehq.apps.fixtures.fixturegenerators.item_lists",
> > -    "corehq.apps.callcenter.fixturegenerators.indicators_fixture_generator",
> > -    "corehq.apps.products.fixtures.product_fixture_generator",
> > -    "corehq.apps.programs.fixtures.program_fixture_generator",
> > -    "corehq.apps.locations.fixtures.location_fixture_generator",
> > -    # custom
> > -    "custom.bihar.reports.indicators.fixtures.generator",
> > -    "custom.m4change.fixtures.report_fixtures.generator",
> > -    "custom.m4change.fixtures.location_fixtures.generator",
> > -]
> > +FIXTURE_GENERATORS = {
> > +    # fixtures that may be sent to the phone independent of cases
> > +    'standalone': [
> > +        # core
> > +        "corehq.apps.users.fixturegenerators.user_groups",
>
> I'm surprised this isn't in the case stage. Seems like case-sharing and
> case-sharing groups are inherently dependent on each other.
>
> Somewhat contrived example: you are moved from one case sharing group to
> another. New cases you create start going to the new group even though you
> still have all the old cases on your phone. The old cases, meanwhile,
> immediately become irrelevant. Is that not a problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/5720/files#r25626104>.
>